### DISCUSSION TITLE
Difficult arenas released!

### DISCUSSION THREAD

**Discussion by fpetric** [larics/icuas22_competition#57] [Announcements]
Dear Teams, 

if you update [larics_gazebo_worlds](https://github.com/larics/larics_gazebo_worlds) and the main competition repository, you can now launch your UAV in two difficult arenas (not the evaluation arenas).

If you manage to navigate through these arenas, your UAV will most likely be able to traverse the evaluation arenas, as they are a bit simpler than these two new arenas. 

To launch the challenge with the new arenas, after updating both [larics_gazebo_worlds](https://github.com/larics/larics_gazebo_worlds) and [icuas22_competition](https://github.com/larics/icuas22_competition/), simply run the challenge with `./start.sh 4` or `./start.sh 5`.

Happy debugging!

**Comment by CatInTheRain** [comment]
Dear organizers,
I did not understand why three days before the official deadline you had the idea of ​​revolutionizing the challenge. Until yesterday I understood that the first phase was about agile flight through obstacles; now the challenge is about exploration in labyrinth with dead ends. I would like to have more explanations regarding this sudden change of course, thanks

**Reply by fpetric** [reply]
Hi @CatInTheRain, 

from the first version of the challenge released on February 1st, the exploration task has been the integral part of the challenge and kept pretty much the same from the start. Directly from the rulebook: 

> To complete the exploration task successfully, the UAV needs to navigate from Zone 1 to Zone 3. ... Team will be provided with bounding boxes of zones 2 and 3 and the UAV will need to find the way through obstacles in Zone 2 to reach Zone 3.

We also explicitly said that the layout of the obstacles is not fixed and may differ. I don't see this as a sudden change of course. Especially since the evaluation arena is not as maze-like as the two we provided for debugging in the most recent update.

**Reply by hinnatariq** [reply]
@fpetric  I second @CatInTheRain  actually, the idea we perceived was the competition is taken from an urban SAR mission (tall buildings not so clustered as in new arena 4 & 5 we see now). Is shouldn't have been so congested to be more like indoor world with passage way. Please if possible look into this and modify the arena complexity. Our team is already stuck in this task and can't see the full arena at our side, its showing with missing parts after even pulling new updates as discussed in thread (https://github.com/larics/icuas22_competition/discussions/55).

**Reply by fpetric** [reply]
As already stated in the reply to @CatInTheRain, the arena we are using to evaluate is not as maze-like as arenas 4 and 5 that we provided for debugging. The reason behind this is, we noticed multiple teams having issues with narrow passages, a series of few pillars that the UAV cannot pass between, especially if the UAV is spawned facing such segment and there is no directly observable passage to Zone 3, replanning the trajectory after additional area is revealed and similar issues that can be resolved if the exploration task is approached as an actual exploration task. You do not have to use them, and your solution may work in the evaluation arena without ever working in these two arenas, especially since we already said that the arena we are evaluating solutions on is not as complex as arenas 4 and 5 (it is closer to arena 3 with respect to complexity).

**Reply by hinnatariq** [reply]
Thanks @fpetric  for the reply and clarification ☺️
Hope we can work out a solution resolving our issues.
